mvvm-test/src/main/java/com/thefuntasty/mvvm/test/ViewModelTest.kt

```diff
@@ -2,6 +2,7 @@ package com.thefuntasty.mvvm.test
 
 import androidx.arch.core.executor.testing.InstantTaskExecutorRule
 import io.github.plastix.rxschedulerrule.RxSchedulerRule
+import kotlinx.coroutines.ExperimentalCoroutinesApi
 import org.junit.Rule
 
 /**
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/mvvm-android/blob/854e22184fc68db6b369995b1ab07bd23e0a7a16/mvvm-test/src/main/java/com/thefuntasty/mvvm/test/ViewModelTest.kt#L8"></span>An abstract class without an abstract member can be refactored to a concrete class.</td>
    </tr>
  </tbody>
</table>